docs/submarine-server/README.md

```diff
@@ -198,6 +198,8 @@ curl -X POST -H "Content-Type: application/json" -d '
 ' http://127.0.0.1:8080/api/v1/jobs      
 ```
 
+For more info about Job RESTful API click [here](./rest-api/job.md).
```

Comment:
Can we put the content of "job.md" to "README.md"? And please remove the "kubectl" way in README.md. That command should be removed since we have the API way to verify and delete jobs.